src/main/java/com/gengo/client/GengoClient.java

```diff
@@ -352,9 +364,147 @@ public JSONObject getTranslationJobs() throws GengoException
      */
     public JSONObject getTranslationJobs(List<Integer> ids) throws GengoException
     {
-        String url = baseUrl + "translate/jobs/";
-        url += join(ids, ",");
-        return call(url, HttpMethod.GET);
+        return this.getTranslationJobs(ids, null, null, null);
+    }
+
+    /**
+     * Get specific translation jobs.
+     * @param ids a list of job ids to retrieve
+     * @param status one of the JobStatus
+     * @param timestampAfter Unix epoch seconds from which to filter submitted jobs
+     * @param count limit count of jobs(default 10, maximum 200)
+     * @return
+     * @throws GengoException
+     */
+    public JSONObject getTranslationJobs(List<Integer> ids, JobStatus status, Integer timestampAfter, Integer count) throws GengoException
```

Comment:
You mean `int count` rather than `interger count`?
I don't understand `need a overload method` for what? Could you please explain it? Thanks.